src/formatter/formatPropValue.js

```diff
@@ -53,6 +53,14 @@ const formatPropValue = (
     )}}`;
   }
 
+  // handle forwardRef and memo
+  if (isPlainObject(propValue) && propValue.$$typeof) {
+    return `{${propValue.displayName ||
+      propValue.type?.name ||
+      propValue.render?.name ||
+      'Component'}}`;
```

Comment:
I would recommend moving this logic to a shared file and reusing it here:
https://github.com/algolia/react-element-to-jsx-string/blob/af5b9d277fbd8de3a9a09d451ede6cfe5ee9768c/src/parser/parseReactElement.js#L34-L45

It would handle more cases, like for example `memo(forwardRef(Component))`